### ISSUE TITLE
Flash message when sending a message is confusing

### ISSUE THREAD

**Issue by saneshark** [department-of-veterans-affairs/messaging-fe#158]
It can take up to 72 hours for a message to be seen? I don't know if it is our responsibility to inform that is it? It should be more clear that we mean a physician might not see the message for 72 hours, but even then, doesn't everyone have the expectation that just because you sent an email does not mean it will be seen right away? 

This to me almost seems like we are saying "our email system is really slow, we deliver messages by carrier pigeon, and it will take 72 hours before it reaches its recipient." And anything else we might be trying communicate is basically akin to, this is email, if you've never used email before you should know, that it takes time for recipient to see the message you sent and reply.

<img width="1022" alt="screen shot 2016-11-07 at 4 54 05 pm" src="https://cloud.githubusercontent.com/assets/591737/20077560/e37dda22-a50a-11e6-9072-2ac1e9b70276.png">

**Comment by gnakm** [comment]
Chiming in. This is something that came up in discovery and it sort of plays as a disclaimer for a user who may get anxious on waiting or may need to escalate their situation in another way. 

We got a suggestion from @suzchap because it is a bit wordy and may it's contributing to the effects that Kam mentioned.

"Your message has been sent. View message. It may take up to 72 hours to receive a response."

**Comment by saneshark** [comment]
How about "Please allow up to 72 hours for a response from your physician"

or "If you have not received a response from your physician within 72 hours, please contact support."

or "If this is a medical emergeny call 911." <-- Unrelated, but I always thought that was a sobering message for me and could work here to prevent escalating when its not necessary.

I'm notoriously bad with copy. These are merely suggestions.

**Comment by ayaleloehr** [comment]
@angel please chime in here with final copy (or redirect as needed!)

**Comment by ahelkit** [comment]
@ayaleloehr @saneshark @gnakm let's go with:

"Your message has been sent. View message. Please allow up to 72 hours for a response from your health care team."

**Comment by ayaleloehr** [comment]
Awesome. Thanks!

Reassigning to @webinista as I believe she implemented the compose message page, which this comes from.
